bindings/rust/bench/benches/throughput.rs

```diff
@@ -67,5 +77,9 @@ pub fn bench_throughput_cipher_suite(c: &mut Criterion) {
     }
 }
 
-criterion_group!(benches, bench_throughput_cipher_suite);
+criterion_group! {
+    name = benches;
+    config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
```

Comment:
What is "100"?
Why is "100"?
😉

Reply:
The tutorial told me so 😬 

I'll add a comment 👍